pages/portal/chapters/create.tsx

```diff
@@ -0,0 +1,152 @@
+import { NextPage } from "next";
+import Head from "next/head";
+import { addChapter } from "requests/Chapter";
+import { Chapter } from 'utils/types';
+
+import Navigation from "components/Portal/Navigation";
+
+const handleSubmit = async (e) => {
```

Comment:
make sure that the parameter e is typed (`any` is fine), otherwise `next build` does not work

Reply:
done!